lib/trino-filesystem-manager/src/main/java/io/trino/filesystem/manager/FileSystemConfig.java

```diff
@@ -18,6 +18,7 @@
 public class FileSystemConfig
 {
     private boolean hadoopEnabled = true;
+    private boolean nativeAlluxioEnabled;
```

Comment:
Let's name this `alluxioEnabled` since we don't have a non-native version. The other `native-` configs are temporary and will be removed later.

Reply:
Renamed to `alluxioEnabled`.